src/index.html

```diff
@@ -3,11 +3,325 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+
+    <link 
+      rel="shortcut icon" 
+      href="images/girl2.png" 
+      type="image/x-icon"
+    >
   </head>
-  <body>
-    <h1>Miami</h1>
+
+  <body class="body" id="home">
+    <div class="container">
+      <header class="header">
+        <nav class="header__nav nav">
```

Comment:
I didn't realise it earlier but for mobile mode the logo and two icons aligned are of smaller size than iPad and desktop mode, please rectify it
![image](https://user-images.githubusercontent.com/100297341/184941594-982aa0d2-dd50-45f8-9c8b-e4a38ea4b984.png)